### ISSUE TITLE
(updated) not working anymore?

### ISSUE THREAD

**Issue by onetimesignup** [EchoDev/TwitchHTML5#8]
if you run browser/twitch themes you may run into what happened to me which was having the player window blocked by an otherwise hidden div, removing it might be enough to get it working again.

not related but if you want you can replace
`// @match        http://twitch.tv/*`
`// @match        http://*.twitch.tv/*`
`// @exclude      http://twitch.tv/*/v/*`
`// @exclude      http://*.twitch.tv/*/v/*`
with
`// @include      /^https?://(.*\.)?twitch\.tv/.*(?!/v/).*$/`

[old] I uninstalled adobe flash to try your script out however it doesn't seem to do anything for us so I signed up to leave you this:

video window error: "No supported video backend available. Flash is not installed"

Firefox: v44
Greasemonkey: v3.6
TwitchHTML5: v1.3

**Comment by EchoDev** [comment]
Is it working for you again? :)

Thanks for the regex, will replace the the code with the include line!

**Comment by onetimesignup** [comment]
I can get it to work by disabling the themes and then re-enable them but it gets tedious, if just removing the div or altering the head tag fixes it then i'd prefer that solution.

Using this fix a little longer made me run into another issue, a certain stream "swectv" ( http://twitch.tv/swectv ) has issues displaying the whole image i get like a smal strip in the middle (see:  http://i.imgur.com/qi2s1WW.png ) and no matter what i do I can't get it to load the stream correctly.

The html5 link works ( http://player.twitch.tv/?channel=$swectv&html5 ) but I have to restart the browser otherwise the bug will carry over to it.

**Comment by EchoDev** [comment]
What themes are you using?

Also I have the same problem on that swectv stream, I wonder what is wrong.

**Comment by EchoDev** [comment]
I have source set as the default quality, when I set the default quality to high the swectv stream works again. I guess that when there is no source available the stream bugs out.

**Comment by onetimesignup** [comment]
I'm currently using a kind of hacky global black theme used by the stylish addon so it seems that when twitch can't find the right window data on the fly it bugs out- I haven't tried it out yet but I figured maybe just remove that overlapping loading div/mask might be all that's needed to display the window correctly.

as for the weird scaling, it might have something to do with the loading gif as well as it's conviently placed in the middle and having pretty much the same width as the viewable strip, I was manipulating the page and the only thing I noticed that had a direct impact on the width was the head tag inside the iframe which also contains width and scaling inside the meta tag.

**Comment by EchoDev** [comment]
If you set the default quality to high you get a proper stream again on swectv. This might be a problem on Twitch.tv's end.

If you can find the div that is blocking your display you can just add this to the script

$(".name-of-div-that-blocks").hide();

It should auto hide it.

**Comment by onetimesignup** [comment]
Yeah high works, it seems like the html5 url autosets it to high while the regular twitch site pushes for source which doesnt work right- I tried with chrome and it has the same issue, i wonder how they managed to break it like this lol.

I'll report back if I manage to solve the load-block thing if not then know that I fought hard =p

**Comment by EchoDev** [comment]
The HTML5 player is still in beta. They might have assumed all streams have Source quality which breaks stuff when there is no source quality available. 

I will be closing the ticket for now, if you need help or something just open it up again :)
